### DISCUSSION TITLE
The resolution to be used in input images for inferencing in a trained nodule detector model

### DISCUSSION THREAD

**Discussion by Rajesh-ParaxialTech** [Project-MONAI/MONAI#7846] [Q&A]
Hello
![image](https://github.com/Project-MONAI/MONAI/assets/159233894/e16e7e03-8ce5-43b9-bed0-8190abc2afef)

In case of a nodule detector model trained with images with resolution 0.7 x 0.7 x 1.25, does that mean the input images used in future during inferencing also must be of the same resolution ?

Thanks

Rajesh

**Comment by KumoLiu** [comment]
Hi @Rajesh-ParaxialTech, Yes, typically, a model trained on images of a specific resolution expects input data in the same resolution during inference. This is because the model learns features and patterns pertinent to the scale and appearance of the training data; changing the resolution can significantly impact the model's performance.
MONAI includes utilities for resampling medical images, which can be used to adjust the spacing of your input images to ensure they match the training data's resolution. This involves resampling the image spatially so that the voxel spacing matches the resolution the network was trained on. Using these utilities can help maintain consistency and allow the model to perform as expected.

Hope it helps, thanks.
